Cargo.toml

```diff
@@ -1,51 +1,21 @@
-[package]
-name = "radix-event-stream"
-version = "1.1.1"
-edition = "2021"
+[workspace]
+members = [
+    "crates/radix-event-stream",
```

Comment:
I think it is better to not have them in the folder crates, but just flat in the directory.